dlclibrary/dlcmodelzoo/modelzoo_download.py

```diff
@@ -20,10 +22,10 @@
     "mouse_pupil_vclose",
     "horse_sideview",
     "full_macaque",
-    "superanimal_topviewmouse",
-    "superanimal_quadruped",
-    "superanimal_quadruped_HRNetw32",
-    "superanimal_quadruped_ATP",
+    "superanimal_topviewmouse_dlcrnet",
```

Comment:
Also, I guess would be better to separate model options from the code. Why? Imagine you will have 20-30 models in the future. It might be better to make a separate config file with available models. what do you think?